src/java/org/apache/cassandra/service/accord/interop/AccordInteropReadRepair.java

```diff
@@ -111,11 +112,11 @@ Object convertResponse(ReadOk ok)
     private static final IVersionedSerializer<Data> noop_data_serializer = new IVersionedSerializer<>()
     {
         @Override
-        public void serialize(Data t, DataOutputPlus out, int version) throws IOException {}
+        public void serialize(Data t, DataOutputPlus out, Version version) throws IOException {}
```

Comment:
even though this could be unversioned, the usage of it uses the same class for replies and that does have a version, so making this unversioned would complicate `ReadDataSerializers.ReplySerializer`